spec/predicates/human-review-crev.md

```diff
@@ -71,26 +77,30 @@ aspects, but the identity of the reviewer can also be mapped based on in-toto's
 functionary handling. `idType` is used to determine the contents of `reviewer`.
 The `url` is a reference to the reviewer's crev-proofs repository.
 
-`reviewTime` _Timestamp_, _required_
+`date` _Timestamp_, _required_
 
 Indicates time of review creation. `timestamp` in the original crev
 specification.
 
 `thoroughness` _enum_, _required_
 
 Describes how thorough the reviewer was. Must be set to one of `low`, `medium`,
-or `high`.
+`high`, or `none`.
```

Comment:
Semantically, what does a `none` value tell a consumer of the attestation? Could this be interpreted as an "endorsement" of a particular package?